### PR TITLE
Reactotron Documentation update

### PR DESCRIPTION
**Pull request by curlyCoder15**
The goal of this PR is to add and update documentation for Reactotron. Some of the main improvements include - 

1. Moving the documentation for plugins from the './apps/plugin-docs' folder to the root docs folder.
2. Deleted the examples in this repo, to make sure there is one true working example "Ignite"
3. Created sections in each document(wherever possible) - `About, Configuration and Usage.`
4. Added file `reactotron-functions.md`, that includes classic functions like `warn, log`
5. Added file `troubleshooting.md` that has information about troubleshooting 
6. Added a file `tips.md`

### CODE REVIEW

**Comment by joshuayoes** [comment]
This isn't changed in the PR, but I noticed that these imports are identical:   https://github.com/infinitered/reactotron/blob/7bd1e240bb69d3f6f710e2bc44cd1ae49ccf75da/docs/plugin-docs/tips.md#piggybacking-on-console
Screenshot 2023-03-14 at 12 04 01 PM

I assume these should be `import Reactotron from 'reactotron-react-native'` and `import Reactotron from 'reactotron-react-js'`

File: README.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ Once you have the application installed it should download and alert you that a
```
Comment (joshuayoes) [inline README.md:26]: Can we add back this section from the current README?

> Use it to:
> 
> view your application state
> show API requests & responses
> perform quick performance benchmarks
> subscribe to parts of your application state
> display messages similar to console.log
> track global errors with source-mapped stack traces including saga stack traces!
> dispatch actions like a government-run mind control experiment
> hot swap your app's state using Redux or mobx-state-tree
> track your sagas
> show image overlay in React Native
> track your Async Storage in React Native
> You plug it into your app as a dev dependency so it adds nothing to your product builds.
> 
> Desktop
> Reactotron on the left, demo React Native app on the right. 

Screenshot 2023-03-14 at 12 07 28 PM

**Review by joshuayoes — COMMENTED** [review]
I have two small nitpicks, but otherwise this looks good!

**Review by silasjmatson — APPROVED** [review]

**Review by joshuayoes — APPROVED** [review]
Looks good! Thanks for making those changes @curlyCoder15

**Pull request merged**